docs/src/content/docs/user-roles/assign-roles.mdx

```diff
@@ -0,0 +1,245 @@
+:::
+
+e.g.
+
+```bash
+curl --request DELETE \
+  --url $CANDIG_URL'/ingest/user/pending' \
+  -H 'accept: application/json' \
+  -H 'Content-Type: application/json' \
+  -H 'Authorization: Bearer '$TOKEN \
+  -d '["user2@test.ca", "user1@test.ca"]'
+```
+
+The response should show that the pending users list is now empty, i.e.:
+
+```bash
+{
+  "pending_users": {}
+}
+```
+
+## Revoke CanDIG Authorized user status <Badge text="Site admin" variant="caution" />
+
+This can only be done by a Site admin. If a user has CanDIG Authorized user status that needs to be revoked, use the `DELETE` method on the user endpoint.
+
+First [get a token](#getting-a-token) following the guide above, then:
+
+```bash
+curl --request DELETE \
+  --url $CANDIG_URL'/ingest/user/user2@test.ca' \
+  -H 'accept: application/json' \
+  -H 'Content-Type: application/json' \
+  -H 'Authorization: Bearer '$TOKEN 
+```
+
+The user will then no longer be able to login and explore the CanDIG data portal.
+
+## Assign the `Site curator` role <Badge text="Site admin" variant="caution" />
+
+This can only be done by a site admin.
+
+Follow the steps in [Getting a Token](#getting-a-token) above then:
+
+1. POST to the `site-role` endpoint in ingest to assign a user the site curator role, e.g. with user1@test.ca
+
+```bash
+curl --request POST \
+  --url $CANDIG_URL'/ingest/site-role/curator/email/user1@test.ca' \
+  -H 'accept: application/json' \
+  -H 'Content-Type: application/json' \
+  -H 'Authorization: Bearer '$TOKEN
+```
+
+2. You can check whether a user has the site curator role by doing the same curl call with a GET request. It should return true.
+ Users can be removed as site curators by using the same endpoint with a DELETE action instead of POST/GET.
+
+
+## Assign `Program Curator` and `Team member` roles <Badge text="Site admin" variant="caution" /> <Badge text="Site Curator" variant="tip" /> <Badge text="Program Curator" variant="success" />
+
+Assigning program curator and team member roles is done through program registration. See the step-by-step guide here: [Registering programs](/CanDIGv2/ingest/register-programs). 
+
+For each program that a curator or team member needs to be added to, a separate program registration will need to be submitted by either a `site admin`, `site curator` or a `program curator` already named on that program.
+
+:::caution
+A `POST` request to the `ingest/program` replaces any existing program registration data for that program. It is advisable to first use a `GET` request to see the current users authorized to a program before adding additional `program_curators` and/or `team_members` when `POST`ing to this endpoint
+:::
+
+## Change the site admin <Badge text="Site admin" variant="caution" />
+
+The site admin can be changed by following the steps on the [Production Deployment page](/CanDIGv2/deployment/production#change-the-default-site-admin).
```

Comment:
added or changed? this seems to imply that there can only be one site admin (is that true??)